modules/network/vpc/variables.tf

```diff
@@ -147,10 +147,33 @@ variable "additional_subnetworks" {
   }
 }
 
+
 variable "secondary_ranges" {
-  type        = map(list(object({ range_name = string, ip_cidr_range = string })))
-  description = "Secondary ranges that will be used in some of the subnets. Please see https://goo.gle/hpc-toolkit-vpc-deprecation for migration instructions."
-  default     = {}
+  type        = list(object({ subnetwork_name = string, ranges = list(object({ range_name = string, ip_cidr_range = string })) }))
```

Comment:
nit. Please use multi-line format. This one-liner is hard to read.

Reply:
Done